Jenkinsfile

```diff
@@ -1,6 +1,4 @@
 // NOTE: Cannot use ACI agents because build requires `unzip` to be available
 buildPlugin(useContainerAgent: true, configurations: [
-        [ platform: "linux", jdk: "8" ],
-        [ platform: "windows", jdk: "8" ],
         [ platform: "linux", jdk: "11" ]
```

Comment:
Java 11 is needed as of 2.361. Tests do not yet pass on Java 17 for this plugin and I did not investigate why.